framework/tests/all-systems/InferTypeArgs2.java

```diff
@@ -1,8 +1,7 @@
 class InferTypeArgs2<V extends InferTypeArgs2<V>> {}
 
 // having a concrete extension of InferTypeArgs2 is the key difference between this and
-// InferTypeArgs1.java.  In this case we end up comparing CFValue with V extends InferTypeArgs2<V>
```

Comment:
This should be reverted. (Rawness here is referring to a generic type without a type argument.)